lib/ash/error/exception.ex

```diff
@@ -13,6 +13,7 @@ defmodule Ash.Error.Exception do
                      [
                        :changeset,
                        :query,
+                       error_context: [],
```

Comment:
the atom `:context` is used by some errors, which cause a compiler error if we try to use `:context` here.
I've chosen `:error_context`, but not completely happy with that name.  Defs open to suggestions.